File: Intermediate/Topic-003-Introduction-to-Arrays/Assigment/Q4-Max-Min-of-an-Array/max-min-of-an-array.c
/**
 * @input A : Integer array
 * @input n1 : Integer array's ( A ) length
 * 
 * @Output Integer
 */
int solve(int* A, int n1) {


    int min = A[0];
    int max = A[0];
    int i;
    for (i = 1; i < n1; i++) {
        if (A[i] < min) {
            min = A[i];
        }
        if (A[i] > max) {
            max = A[i];
        }
    }

    int total = min + max;
    return total;


}

File: Intermediate/Topic-003-Introduction-to-Arrays/Assigment/Q4-Max-Min-of-an-Array/max-min-of-an-array.py
class Solution:
    # @param A : list of integers
    # @return an integer
    def solve(self, A):
        min = A[0]
        max = A[0]

        for i in range(len(A)):
            if A[i] < min:
                min = A[i]
            if A[i] > max:
                max = A[i]
        total = max + min
        return total

